config/locales/time/en.yml

```
@@ -2,6 +2,7 @@
 en:
   date:
     day_names:
+      -
```

adding this line allows `%A` to grab the correct day of the week, versus previously when it was giving the name of the next day

---

should we add that as a YAML comment?
```suggestion
      - # empty item for correct weekday offset for %A
```